content/v0.80.0.md

```diff
@@ -0,0 +1,478 @@
+ 
+**Dual mouse gaming:**
+
+Multiple physical mice can now be mapped to specific emulated mouse interfaces,
+enabling two-player split-screen gaming (The Settlers, The Settlers II) using
+two USB mice on a single computer.
+
+<video controls width=640>
+  <source src="/dual_mouse_s2.mp4" type="video/mp4">
+  Your browser does not support the <code>video</code> element.
+</video>
+
+See the (wiki article)[https://github.com/dosbox-staging/dosbox-staging/wiki/Dual-Mouse-Gaming] for more details.
+
+
+**Improved support for 3rd party INT33 Windows mouse driver:**
+ 
+@javispedro's (INT33 mouse
+driver)[https://git.javispedro.com/cgit/vbados.git/about/] for Windows 3.1x is
+fully supported and no longer requires the use of the separate DOS driver,
+except when booting from a disk image. The driver also resolves compatibility
+issues with seamless mouse integration and adds mouse wheel support.
+
+See the (wiki article)[https://github.com/dosbox-staging/dosbox-staging/wiki/Windows-3.1x#improved-mouse-driver] for more details.
+
+**Mouse-related help in the title bar:**
+
+The title bar now displays some short help about the mouse, such as how it can
+be captured or released.
+
+###  4.3. <a name='AbilitytouselocalconfigsviaFinderonmacOS'></a>Ability to use local configs via Finder on macOS
+
+A new `--working-dir <path>` command line parameter has been introduced to make
+it possible to launch DOSBox Staging using local configs from the Finder on
+macOS. Previously, you had to use DOSBox from the command line to achieve this.
+ 
+**How to use it:**
+
+Follow the steps indicated in the DMG package, as shown: 
+
+[![macOS DMG run-command tip](/macos_dmg_run_command.png)](/dmacos_dmg_run_command.png)
+
+Simply copy the included `Start DOSBox Staging.command` to any folder that
+contains a local `dosbox.conf` and then open it (`Start DOSBox Staging.command`)
+in Finder. DOSBox Staging will act as if started from that folder and will pick
+up the local config.
+ 
+Note: you might be asked to give DOSBox Stating access to the containing folder
+upon the first launch — just click OK if that dialog comes up.
+
+ 
+##  5. <a name='Generalimprovements'></a>General improvements
+
+###  5.1. <a name='more.comcommand'></a>`more.com` command
+
+The `more` command is now internally implemented (should be more user friendly
+than the original DOS one), and its engine is used for displaying help messages
+longer than one screen.
+
+<video controls width=640>
+  <source src="/enhanced_more.mp4" type="video/mp4">
+  Your browser does not support the <code>video</code> element.
+</video>
+
+###  5.2. <a name='On-the-flychanges'></a>On-the-fly changes
+
+Several settings can be safely adjusted at runtime, including the output video
+settings, resolution, GL shaders, audio devices, and MIXER settings:
+
+<video controls width=640>
+  <source src="/plug_n_play_audio.mp4" type="video/mp4">
+  Your browser does not support the <code>video</code> element.
+</video>
```

Comment:
This won't show the video in all browsers. We had this problem with the 0.79.0 release notes, Safari wouldn't show anything if I remember correctly.

This is how we solved it, so you'll need to provide the video in `.webm` format as well. I think you came up with the ffmpeg command for generating the webm, might be still pinned in one of the channels :) 

```html
<video controls width=640>
  <source src="/pcspeaker_impulse_wizball.webm" type="video/webm; codecs=vp9,opus">
  <source src="/pcspeaker_impulse_wizball.mp4" type="video/mp4">
  Your browser does not support the <code>video</code> element.
</video>
```

Reply:
h.264-encoded mp4's should be everything-compatible.